Makefile

```diff
@@ -95,6 +96,10 @@ flake8: install-dev
 tests: version-file
 	$(VENV)/bin/tox
 
+tdd: install-dev
+	$(VENV)/bin/pytest-watch
```

Comment:
Do you mean to install pytest-watch here? In that case you should use ``$(VENV)/bin/pip install pytest-watch``